website/src/pages/dictionary/index.js

```diff
@@ -22,11 +22,15 @@ import startCase from 'lodash/startCase';
 import get from 'lodash/get';
 import { TAG_TYPES } from '../../components/Tag';
 import { format as formatDate } from 'date-fns';
-import flatten from 'lodash/flatten';
-import ReactDOM from 'react-dom';
```

Comment:
Kinda need this 😝

Reply:
@joneubank my bad. was removing dups.  threw the baby out with the bathwater.